src/airport.js

```diff
@@ -1 +1,59 @@
+class Airport {
```

Comment:
Good part about the code

It's organized into a class, encapsulating related properties and methods within a single entity. This promotes code reusability and maintainability.

It's relatively simple and easy to understand due to simplicity and variable and function names are descriptive and provide a clear understanding of their purpose and functionality. T